resource-watch/resource-watch
# Next@9.1.1

## Overview
Updates app to the latest version of `next` (`@9.1.1`).

More info: https://nextjs.org/blog/next-9-1

Removed `<Container>` Next's component as is deprecated: https://github.com/zeit/next.js/blob/master/errors/app-container-deprecated.md

Moved `static` folder to `public/static`: https://nextjs.org/blog/next-9-1#public-directory-support

For https://github.com/resource-watch/resource-watch/pull/1161/commits/04c15a16bf8a8ea5a06c9560e2a0e0c92a467c2c, don't forget to update in your `.env` `NODE_ENV` to `RW_NODE_ENV`. I was forced to make this change due to https://github.com/zeit/next.js/blob/master/errors/env-key-not-allowed.md


## Testing instructions
`rm -rf node_modules && yarn`.

Check app initializes and works properly. Pay attention to files loaded from `static` folder (like widget editor images, Cesium, etc.) and make sure it loads. This update is according to https://nextjs.org/docs#build-time-configuration

## Pivotal task
–


---

## Checklist before submitting
